Assignment8_FirstCommit

GobindaFexleTrainee opened the pull request:
Added Apex classes,objects,tabs and layouts

GobindaFexleTrainee committed:
Assignment8_First_Commit

Added Apex classes,objects,tab and layout

GobindaFexleTrainee committed:
Update package.xml

GobindaFexleTrainee committed:
Update Insert_Student_Details.cls

GobindaFexleTrainee committed:
Update StudentQuery.cls

shubhamsharmafexletraining commented on src/classes/Insert_Student_Details.cls:
```diff
@@ -0,0 +1,50 @@
+        student.City__c='Kolkata';
+        student.State__c='WestBengal';
+	insert student;
+        System.Debug('Inserted Record id :'+student);
```
to get Id, use the student.Id instead student.

shubhamsharmafexletraining replied:
Please correct this.

shubhamsharmafexletraining commented on src/classes/StudentQuery.cls:
```diff
@@ -0,0 +1,9 @@
+public class StudentQuery {
+    public StudentQuery(){
+		List<Student__c> studentList = [select id, name, Roll_Number__c ,Mobile_Number__c,Father_Name__c,Father_Mobile_Number__c,
+		HomePhone__c,Email__c,Address__c,Overall_Grade__c from Student__c order by createdDate DESC LIMIT 5];
```
Add all the fields into the query.

shubhamsharmafexletraining commented on src/package.xml:
```diff
@@ -1,20 +1,79 @@
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
     <types>
-        <members>DreamWorld</members>
+        <members>Student</members>
```
Add the correct class name.

shubhamsharmafexletraining commented on src/package.xml:
```diff
@@ -1,20 +1,79 @@
     <types>
-        <members>Dream_World__c.Dream_World_Title__c</members>
-        <name>CustomField</name>
+        <members>StudentSubjects__c.Student__c</members>
```
Do not require adding custom fields if you are also adding that custom object.

shubhamsharmafexletraining reviewed (changes requested):
Fix the PR comments

GobindaFexleTrainee committed:
Assignment8_2nd_commit

removes custom fields,adding custom fields in Studentquery and corrected class names

GobindaFexleTrainee committed:
Merge branch 'Assignment8' of https://github.com/GobindaFexleTrainee/SF-TRAINING2 into Assignment8

shubhamsharmafexletraining commented on src/classes/StudentQuery.cls:
```diff
@@ -0,0 +1,8 @@
+public class StudentQuery {
+    public StudentQuery(){
+		List<Student__c> studentList = [select id, name,StudentSubjects__c,Account_Number__c,Address__c,Bank_Name__c,Bank_Phone_Number__c,City__c,Class__c,College_Address__c,College_Location__c,Current_Year__c,DOB__c,Donation_Amount__c,Email__c,Exam_Fees_Submitted__c,Father_Mobile_Number__c,Father_Name__c,Favourite_Subjects__c,Fees_Submitted__c,HomePhone__c,IFSC_Code__c,Is_Donation__c,Mobile_Number__c, ...
```
Add line break and use correct indentation for each line.

shubhamsharmafexletraining reviewed (changes requested):
Fix remaining comments.

GobindaFexleTrainee committed:
Update StudentQuery.cls

breaks Query in lines

shubhamsharmafexletraining commented on src/classes/StudentQuery.cls:
```diff
@@ -0,0 +1,14 @@
+public class StudentQuery {
+    public StudentQuery(){
+		List<Student__c> studentList = [select id, name,StudentSubjects__c,Account_Number__c,Address__c,Bank_Name__c,Bank_Phone_Number__c,City__c,Class__c,
```
This is what correct indentation looks like for this:

![image](https://user-images.githubusercontent.com/106371854/173565236-f3d6b1be-7f85-49d6-b1b9-a42bc5d29be4.png)

shubhamsharmafexletraining replied:
Please Correct this.

GobindaFexleTrainee committed:
Assignment_8_third_commits

changed   student to student.id in insert_student_details class

GobindaFexleTrainee committed:
Merge branch 'Assignment8' of https://github.com/GobindaFexleTrainee/SF-TRAINING2 into Assignment8

GobindaFexleTrainee committed:
Update StudentQuery.cls

shubhamsharmafexletraining reviewed (changes requested):
Please fix the PR comment for StudentQuery.cls

GobindaFexleTrainee committed:
Update StudentQuery.cls

GobindaFexleTrainee committed:
Update StudentQuery.cls

GobindaFexleTrainee committed:
Update StudentQuery.cls

shubhamsharmafexletraining commented on src/objects/Student__c.object:
```diff
@@ -0,0 +1,863 @@
+    <recordTypes>
+        <fullName>Register</fullName>
+        <active>false</active>
+        <label>Register</label>
```
Correct the RecordType name. It should be "Registrar"

shubhamsharmafexletraining reviewed (changes requested):
Fix the RecordType Name for Registrar

GobindaFexleTrainee committed:
Update Student__c.object

changed register to Registrar

shubhamsharmafexletraining reviewed (approved):
Approved
